docs/libertyDev.md

```diff
@@ -99,7 +99,7 @@ test {
 
 Start a Liberty server in a local container using the Dockerfile that you provide. An alternative to the `libertyDevc` task is to specify the `libertyDev` task with the `--container` option. 
 
-You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Gradle task. The installed Docker Engine version must be 18.03.0 or higher.
+You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Gradle task. The installed Docker Client and Engine version must be 18.03.0 or higher.
```

Comment:
Should this be plural for `The installed Docker Client and Engine versions`?

Reply:
Yes, that reads better.